docs/expected_loss.ipynb

```diff
@@ -0,0 +1,203 @@
+      "text/plain": [
+       "Graphics object consisting of 5 graphics primitives"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Optimal federation sizes, details\n",
+    "graph_fedsizes_baseline([i * 3 + 1 for i in range(1, 5)], pmax=0.1, title=\"Optimal federation sizes\")"
```

Comment:
This graph shows the low malicious fraction part of the chart pretty well, but it might be nice to also see the crossover points with the "split over n" line. Those points could be graphed here or on another graph, or solved for numerically and presented in a table.

Reply:
I added a zoomed in version. The concrete values are less relevant imo since they are so huge that no one should use a custodian at that point imo.